src/common/util/report-bug.js

```diff
@@ -0,0 +1,64 @@
+const http = require('http');
+const https = require('https');
+
+const { getConfigValue } = require('homegames-common');
+
+const ERROR_REPORTING_ENABLED = getConfigValue('ERROR_REPORTING', false);
+const HTTPS_ENABLED = true;//'https://api.homegames.io/bugs';//getConfigValue('HTTPS_ENABLED', false);
```

Comment:
https is now enabled by default. http is still default when running homegames core/web